plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -114,7 +114,9 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
                 prop: keys.port,
                 className: getColumnClassNameFn(),
                 render(prop, row) {
-                  return getDisplayValue(row[prop]);
+                  return appDefinition.requirePorts
+                    ? getDisplayValue(row[prop])
+                    : <em>Auto Assigned</em>;
```

Comment:
@bstavroulakis can you confirm `em` (emphasis) was approved by design please? other than that works perfectly.